packages/testing/ts/__tests__/e2e.test.ts

```diff
@@ -1476,12 +1466,14 @@ describe("e2e tests", function test() {
     it("join the second and third polls", async () => {
       // eslint-disable-next-line @typescript-eslint/prefer-for-of
       for (let pollId = 1; pollId <= 2; pollId += 1) {
-        for (let i = 0; i < users.length; i += 1) {
```

Comment:
You can keep this

Reply:
![image](https://github.com/user-attachments/assets/1dc00241-8c18-4dce-ab5b-a36be89ef72a)

I am getting the es-lint error :S I can see that we are using `// eslint-disable-next-line @typescript-eslint/prefer-for-of` on the for above, I could use that one as well but please check Ale's comment to recommend me which es-lint silencer we use

Reply:
We have this code working in other tests with for loop and eslint ignore comment (`eslint-disable-next-line @typescript-eslint/prefer-for-of`)